en/includes/guides/notification-channels/configure-email-provider.md

```diff
@@ -0,0 +1,101 @@
+          <br/> For <code>BASIC</code> you need to configure <code>username</code> and <code>password</code>. 
+          <br/> For <code>CLIENT_CREDENTIAL</code> you need to configure <code>client_id</code>, <code>client_secret</code>, <code>token_endpoint</code> and  <code>scopes</code>.
+          Support for the <code>CLIENT_CREDENTIAL</code> authentication type is available for Microsoft 365 Exchange Online.
+        </td>
+      <tr>
+        <td><code>username</code></td>
+        <td>Provide the username of the SMTP account. <br/> Username of the mail you have provided in <strong>from_address</strong>.</td>
+      </tr>
+      <tr>
+        <td><code>password</code></td>
+        <td>Provide the password of the SMTP account. <br/> Password of the mail you have provided in <strong>from_address</strong>.</td>
+      </tr>
+      <tr>
+        <td><code>client_id</code></td>
+        <td>Provide the password of the SMTP account. <br/> Password of the mail you have provided in <strong>from_address</strong>.</td>
+      </tr>
+      <tr>
+        <td><code>host</code></td>
+        <td>The SMTP server to connect to.</td>
+      </tr>
+      <tr>
+        <td><code>port</code></td>
+        <td>This is the SMTP server port to connect to if the connect() method does not explicitly specify one. Defaults to 25.</td>
+      </tr>
+      <tr>
+        <td><code>enable_start_tls</code></td>
+        <td>If true, this enables using the <code>STARTTLS</code> command (if enabled before issuing any login commands. Note that an appropriate trust store must be configured so that the client will trust the server's certificate. Defaults to <code>true</code>.</td>
+      </tr>
+      <tr>
+        <td><code>enable_authentication</code></td>
+        <td>If true, attempt to authenticate the user using the AUTH command. Defaults to <code>true</code>.</td>
+      </tr>
+      <tr>
+        <td><code>signature</code></td>
+        <td>Signature of the sender account.</td>
+      </tr>
+    </table>
+
+    !!! Tip
+        For information about the SMTP, see 
+        [here](https://javaee.github.io/javamail/docs/api/com/sun/mail/smtp/package-summary.html){:target="_blank"}.
+
+    !!! info
+        - If you use a Gmail account as the **from_address**, you must create an [App Password](https://support.google.com/accounts/answer/185833?visit_id=637943607149528455-3801902236&p=InvalidSecondFactor&rd=1){:target="_blank"}.
+         After you get an **App Password** from Google, update the `password`.
+        - If your password contains special characters (example: `<`, `>`, `&`), you will run into errors when running the server. To avoid errors, update the `password` parameter as follows:
+        ```toml
+        password= "<![CDATA[xxxx]]>"
+        ```
+
+3. Save the configurations and start the server.
+
+## Configure the email provider (per tenant)
```

Comment:
Maybe we need to say that super tenant email provider configurations cannot be performed through console